translations/hu/get5.phrases.txt

```diff
@@ -0,0 +1,379 @@
+    }
+    "AdminForcePauseInfoMessage"
+    {
+        "hu"            "Egy admin, szüneteli a mérkőzést."
+    }
+    "AdminForceUnPauseInfoMessage"
+    {
+        "hu"            "Egy admin, a mérkőzés folytatását hajtotta végre."
+    }
+    "TeamWantsToReloadLastRoundInfoMessage"
+    {
+        "hu"            "{1} szeretne megállni és visszatölteni az előző kört. {2} el kell fogadni a {GREEN}!stop {NORMAL}paranccsal."
+    }
+    "TeamWinningSeriesInfoMessage"
+    {
+        "hu"            "{1}{NORMAL} áll nyerésre {2}-{3}"
+    }
+    "SeriesTiedInfoMessage"
+    {
+        "hu"            "Az állás döntetlen. {1}-{2}"
+    }
+    "NextSeriesMapInfoMessage"
+    {
+        "hu"            "A következő pálya a szériában {GREEN}{1}"
+    }
+    "TeamWonMatchInfoMessage"
+    {
+        "hu"            "{1} megnyerte a mérkőzést."
+    }
+    "TeamTiedMatchInfoMessage"
+    {
+        "hu"            "{1} és a {2} döntetlent játszott."
+    }
+    "TeamsSplitSeriesBO2InfoMessage"
+    {
+        "hu"            "{1} és a {2} kiegyenlítettek. 1-1-re."
+    }
+    "TeamWonSeriesInfoMessage"
+    {
+        "hu"            "{1} nyerte a szériát. {2}-{3}"
+    }
+    "MatchFinishedInfoMessage"
+    {
+        "hu"            "A mérkőzés befejeződött"
+    }
+    "CurrentScoreInfoMessage"
+    {
+        "hu"            "{LIGHT_GREEN}{1} {GREEN}{2} {NORMAL}- {GREEN}{3} {LIGHT_GREEN}{4}"
+    }
+    "StopCommandNotEnabled"
+    {
+        "hu"            "A stop parancs nincs engedélyezve."
+    }
+    "BackupLoadedInfoMessage"
+    {
+        "hu"            "Sikeresen visszatöltődött a körmentés {1}"
+    }
+    "MatchBeginInSecondsInfoMessage"
+    {
+        "hu"            "A mérkőzés {1} másodperc múlva kezdődik."
+    }
+    "MatchIsLiveInfoMessage"
+    {
+        "hu"            "A mérkőzés {GREEN}ÉLES"
+    }
+    "KnifeIn5SecInfoMessage"
+    {
+        "hu"            "A kés kör 5 másodperc múlva kezdődik."
+    }
+    "KnifeInfoMessage"
+    {
+        "hu"            "Kés!"
+    }
+    "TeamDecidedToStayInfoMessage"
+    {
+        "hu"            "{1} úgy döntött, hogy maradnak."
+    }
+    "TeamDecidedToSwapInfoMessage"
+    {
+        "hu"            "{1} úgy döntött, hogy térfelet cserélnek."
+    }
+    "TeamLostTimeToDecideInfoMessage"
+    {
+        "hu"            "{1} maradnak, mivel idő letelte előtt, nem választottak."
+    }
+    "ChangingMapInfoMessage"
+    {
+        "hu"            "Pályaváltás a következőre {GREEN}{1}..."
+    }
+    "MapDecidedInfoMessage"
+    {
+        "hu"            "A pályák ellettek döntve:"
+    }
+    "MapIsInfoMessage"
+    {
+        "hu"            "{1}. Pálya: {GREEN}{2}{NORMAL}"
+    }
+    "TeamPickedMapInfoMessage"
+    {
+        "hu"            "{1} választotta a {GREEN}{2} {NORMAL} {3}. pályának."
+    }
+    "TeamSelectSideInfoMessage"
+    {
+        "hu"            "{1} választott, hogy kezd {GREEN}{2} {NORMAL} oldalon a {3}. pályán"
```

Comment:
Why "pályán" after the full stop?

Reply:
Ah, sorry, I'm an idiot. The full stop is the n-th indicator. But it needs a full stop after "pályán", no?

Reply:
{1} - team , {2} - side, {3} - map.
{1} decided to start {2} side on {3} map
XYZ decided to start CT side on First map.
in Hungarian: XYZ választott, hogy kezd CT oldalon a 1. pályán.
the full stop means the n-th, yeah.

Reply:
But it needs a full stop after "pályán", no?
- Yes, sorry wasn't got my attention